<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Brain_Regions_Panel</class>
 <widget class="QWidget" name="Brain_Regions_Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="CardWidget" name="CardWidget">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="TitleLabel" name="TitleLabel">
          <property name="text">
           <string>Brain Regions</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="PrimaryPushButton" name="select_all_PrimaryPushButton">
            <property name="text">
             <string> Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PrimaryPushButton" name="clear_all_PrimaryPushButton">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="BrainRegionsWidget" name="brain_regions_widget" native="true"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BrainRegionsWidget</class>
   <extends>QWidget</extends>
   <header>zjb.gui.widgets.brain_regions_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
